tests/core/editable/manual/enterbr.md

```diff
@@ -0,0 +1,20 @@
+@bender-tags: 4.16.3, bug, 3858
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, clipboard, enterkey, format,
+
+
+1. Open browser's console.
+2. Copy two paragraphs placed above the editor. `<p>a</p><p>b</p>` and paste into editable.
+3. Hit enter twice.
+4. Type any ( only one ) letter and remove it using `backspace`.
+5. Paste the the same two paragraphs from `step 1` ( don't change cursor position ).
+
+**Expected**
+ * The paragraphs from `step 5` are pasted correctly.
+ * The indentation is correct.
```

Comment:
The expectations should be very clear. `Correct` therm sometimes could be interpreted differently by two different persons. I assume, that here, we need something like:`Pasted paragraphs are nor indented` or sth similar?

Also, we need additional mention that paragraphs pasted in step 5 are starting from the cursor position, so no lines were 'eaten' after the paste.